fossunited/foss_hackathon/doctype/foss_hackathon_localhost/foss_hackathon_localhost.py

```diff
@@ -35,55 +35,70 @@ class FOSSHackathonLocalHost(Document):
     pass
 
     def before_insert(self):
-        self.assign_localhost_organizer_role()
+        members = [member.profile for member in self.organizers]
```

Comment:
design question : Why do even need the `FOSSHackathonLocalHostOrganizer` doctype? Why not just use `FOSSUserProfile`?

```
organizers: DF.Table[FOSSUerProfile]
```

From the looks of it, the only meaningful info on `FOSSHackathonLocalHostOrganizer` is `FOSSUserProfile` - which surprisingly can be `None` as well which I think might cause subtle bugs.

Reply:
`FOSS Hackathon LocalHost Organizer` is a child doctype, which is just a fancy way to introduce tables to a "parent" doctype. This doctype lets us add a list of profiles to a localhost as organizers
![image](https://github.com/user-attachments/assets/0adedfcf-fece-4277-b462-43f009f4d425)

I don't think it is possible to just add profile to a doctype. You need to create a child table with a profile doctype